_posts/2016-05-23-opaque-tokens-in-angular-2.md

```diff
@@ -44,15 +44,15 @@ class MyComponent {
 {% endraw %}
 {% endhighlight %}
 
-Or, in this particular case, we can also use the shorthand version, as the token matches the dependency instance type and the provider strategy is `useClass`:
+Since the token matches the dependency instance-type and the provider strategy is `useClass`, we can actually also the **shorthand** version, as:
 
 {% highlight js %}
 {% raw %}
 providers: [DataService]
 {% endraw %}
 {% endhighlight %}
 
-That's cool, as long as we have classes (or types) to represent the things we want to work with. However, sometimes we need to create other objects that don't necessarily need to be put in a class representation. We could for example have a configuration object that we want to inject into our application. This configuration object can be a simple object literal, there's no type involved.
+Angular2 has many shorthand versions; and ^ is just one of those. Now this is cool, as long as we have classes (or types) to represent the things we want to work with. However, sometimes we need to create other objects that don't necessarily need to be put in a class representation. We could for example have a configuration object that we want to inject into our application. This configuration object can be a simple object literal where there is no type involved.
```

Comment:
That's new to me. Actually there's only one shorthand, which is the one showed. Everything that doesn't match that rule needs to go with the more verbose syntax.